### PR TITLE
Popup keyboard navigation bug when dismissing popup

### LINKED ISSUE #681: Popup keyboard navigation bug when dismissing popup
When you dismiss a popup with "Escape" key or by tabbing to and activating the "X" affordance, the focus should fall back on the feature that is being interrogated, and a tooltip should be displayed / read.  While focus usually goes to the feature, no tooltip is displayed.

### PR DESCRIPTION
**Pull request by kevinkim31**
Closes #681

### CODE REVIEW

**Comment by kevinkim31** [comment]
commits [349d388](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/726/commits/349d38803700445e6569a3e38afa3c20c85eb7e3) , [57e710b](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/726/commits/57e710b17d8ebfc8e22a20b79eafd3372c05d394) , [0e9614e](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/726/commits/0e9614ebf996d82c5a0fa641a6177ea751d69134) were just to keep web-map-cuistom-element changes updated with local. These changes don't affect code (No files changed from these commits).

**Comment by prushforth** [comment]
I noticed and maybe it's a different bug or perhaps a regression, that when you use the arrow keys to "tab" into a set of features, then you popup a dialog and dismiss it, either with esc or X, if you hit "Tab" then you don't move past the features, you move to the next feature.  That tab should take you to the zoom out button, shouldn't it?  Try it on [experiments](https://maps4html.org/experiments/screenreader/restaurants/), although because of this bug it's hard to say if it's a part of this bug or another bug??

File: src/mapml/features/featureGroup.js [hunk 1]
```diff
@@ -98,9 +98,10 @@ export var FeatureGroup = L.FeatureGroup.extend({
       this._map.featureIndex.currentIndex = 0;
       this._map.featureIndex.inBoundFeatures[0].path.focus();
     }
-
+  
+    // 27 added so that the tooltip opens when dismissing popup with 'esc' key
     if(e.target.tagName.toUpperCase() !== "G") return;
-    if(([9, 13, 16, 37, 38, 39, 40, 49, 50, 51, 52, 53, 54, 55].includes(e.keyCode)) && e.type === "keyup") {
+    if(([9, 13, 16, 37, 38, 39, 40, 49, 50, 51, 52, 53, 54, 55, 27].includes(e.keyCode)) && e.type === "keyup") {
```
Comment (AliyanH) [inline src/mapml/features/featureGroup.js:104]: Great work on this PR, it's working like a charm. I just had some trouble with the test for this change you made ("_Tooltip appears after pressing esc key_"). When the 27 keycode is removed from this line, the test for it still passes.

File: test/e2e/core/popupTabNavigation.test.js [hunk 2]
```diff
@@ -127,6 +127,47 @@ test.describe("Playwright Keyboard Navigation + Query Layer Tests" , () => {
       expect(f).toEqual("M330 83L586 83L586 339L330 339z");
     });
 
+    test("Tooltip appears after pressing esc key", async () => {
+      await page.keyboard.press("Enter"); 
+      await page.waitForTimeout(500);
+      await page.keyboard.down("Escape"); // focus back on feature
+      await page.waitForTimeout(500);
```
Comment (AliyanH) [inline test/e2e/core/popupTabNavigation.test.js:134]: don't think timeout is needed here

**Review by AliyanH — APPROVED** [review]
LGTM!

**Review by prushforth — APPROVED** [review]
LGTM! Thanks everyone for having a look together, it helps.

**Pull request merged**